fs.go

```diff
@@ -75,6 +75,14 @@ func writeFile(path string, in toml.Marshaler) error {
 	return err
 }
 
+// modifyWithString - Modifies a given file with a new string input.
```

Comment:
nit: s/modifyWithString - Modifies/modifyWithString modifies/

Just slightly more standard practice for docblocks.